lib/gtag.js

```diff
@@ -0,0 +1,16 @@
+import { gaTrackingId } from "constants/env";
+
+// https://developers.google.com/analytics/devguides/collection/gtagjs/pages
+export const pageview = url => {
+  window.gtag("config", gaTrackingId, {
+    page_location: url
+  });
+};
+
+// https://developers.google.com/analytics/devguides/collection/gtagjs/events
+export const event = gaEvent => {
+  window.gtag("event", gaEvent.contributor, {
+    category: `${gaEvent.type} : ${gaEvent.partner}`,
+    label: `${gaEvent.itemId} : ${gaEvent.title}`
```

Comment:
In the docs/examples you link to in the PR, the keys are `event_category` and `event_label` rather than `category` and `label`.  I'm guessing this makes a difference?

Reply:
correct. changed (re-sanity check?). i notice you don't use the `value` and assume that's fine